server/index.js

```diff
@@ -9,8 +9,8 @@ app.use('/:gameId', express.static('public'));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 
-
 app.get('/api/overview/:gameId', (req, res) => {
+  // console.log('req.params.gameId', req.params.gameId);
```

Comment:
Maybe throw in some error handling? (Or otherwise, maybe write a jest test to ensure your get request is functioning properly.)